types/vue.d.ts

```diff
@@ -80,10 +80,28 @@ declare namespace NuxtVueI18n {
  * Extends types in vue
  */
 declare module "vue/types/vue" {
+  interface NuxtI18nSeo {
```

Comment:
I don't really know much about typescript or writing declaration files but should this interface be defined outside of the `declare module` block?
@kevinmarrec

Reply:
Let's merge this as is for now.
@kevinmarrec If you get the chance to have a look at this, could you answer @rchl's question?

Reply:
@paulgv Sorry I was away from keyboard.

@rchl Is right, the interface definition should be before declaring module and not inside.